packages/apps/public/locales/ru/translation.json

```
@@ -165,7 +164,7 @@
   "If the recipient account is new, the balance needs to be more than the existential deposit. Likewise if the sending account balance drops below the same value, the account will be removed from the state.": "Если получатель - новый аккаунт, то баланс должен быть больше минимального депозита. Аналогично если баланс аккаунта отправителя упадет ниже этого значения, то аккаунт будет исключен из состояния.",
   "If this proposal is passed, the changes will be applied via dispatch and the deposit returned.": "Если это предложение будет одобрено, то изменения будут внесены и депозит возвращается",
   "Important notice": "Важное уведомление",
-  "In calculating the election outcome, this prioritized vote ordering will be used to determine the final score for the candidates.": "Для подсчета результатов голосования будет использовано приоретизированное голосование",
+  "In calculating the election outcome, this prioritized vote ordering will be used to determine the final score for the candidates.": "Для подсчета результатов голосования будет использовано приоритезированное голосование",
```

```suggestion
  "In calculating the election outcome, this prioritized vote ordering will be used to determine the final score for the candidates.": "Для подсчета результатов голосования будет использовано приоритизированное голосование",
```
https://ru.wiktionary.org/wiki/%D0%BF%D1%80%D0%B8%D0%BE%D1%80%D0%B8%D1%82%D0%B8%D0%B7%D0%B8%D1%80%D0%BE%D0%B2%D0%B0%D1%82%D1%8C